tests/sentinel/tmp/.gitignore

```diff
@@ -1,2 +1 @@
-redis_*
-sentinel_*
```

Comment:
sentinel definetly need an update, i once doing rebase and add a lot of sentinel/tmp/valkey_xxx files after a sentinel test.